presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -963,7 +963,9 @@ else if (criteria instanceof JoinOn) {
                 // we do it further down when after we determine which subexpressions apply to left vs right tuple)
                 ExpressionAnalyzer analyzer = ExpressionAnalyzer.create(analysis, session, metadata, sqlParser, accessControl);
```

Comment:
In fact can be removed.
Changed in commit 70daa0f50da949cdd3cc7cdcec51f188d382d457 (supersedes commit 33e2a8122c76ca093fbd129d012cce86b533e411).